src/App.js

```diff
@@ -544,53 +581,55 @@ class App extends React.Component {
       return urls;
     });
   };
+  //#endregion
 
   render() {
     return (
-      <Router>
-        <ScrollToTop>
-          <div className="flyout">
-            {!this.state.caching &&
-              this.state.fetchedUser &&
-              this.state.loggedUser?.username ===
-                this.state.fetchedUser.platformData.user?.username && (
-                <MDBProgress material preloader className="caching-loader" />
-              )}
-            <Navbar
+      <ScrollToTop>
+        <div className="flyout">
+          {!this.state.caching &&
+            this.state.fetchedUser &&
+            this.state.loggedUser?.username ===
+              this.state.fetchedUser.platformData.profile?.username && (
+              <MDBProgress material preloader className="caching-loader" />
+            )}
+          <Navbar
+            globalState={this.state}
+            globalFunctions={{
+              logout: this.logout,
+              saveSettings: this.saveSettings,
+              users: this.getAllPageUrls,
+            }}
+          />
+          <main>
+            <Routes
               globalState={this.state}
               globalFunctions={{
-                logout: this.logout,
-                saveSettings: this.saveSettings,
-                users: this.getAllPageUrls,
+                fetchCacheData: this.fetchCacheData,
+                updateCache: this.updateCache,
+                uploadTalk: this.uploadTalk,
+                deleteTalk: this.deleteTalk,
+                getTalk: this.getTalk,
+                login: this.login,
+                registerUser: this.registerUser,
+                fetchGitLabServers: this.fetchGitLabServers,
+                refetchRequired: this.refetchRequired,
+                usernameMatchesFetchedUsername: this
+                  .usernameMatchesFetchedUsername,
```

Comment:
Does this pass the max line length or why is this written in two lines?
```suggestion
                usernameMatchesFetchedUsername: this.usernameMatchesFetchedUsername,
```

Reply:
Yep your prediction is completely right.